docs/template_plugin/tests/functional/op_reference/topk.cpp

```diff
@@ -0,0 +1,1547 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "openvino/op/topk.hpp"
+#include "openvino/op/constant.hpp"
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ov;
+
+namespace {
+struct TopKParams {
+    TopKParams(
+        const Tensor& A, const Tensor& k, const int64_t axis,
+        const op::v1::TopK::Mode mode, const op::v1::TopK::SortType sort,
+        const Tensor& result0, const Tensor& result1,
+        const std::string& testcaseName = "") :
+        A(A), k(k), axis(axis), mode(mode), sort(sort),
+        result0(result0), result1(result1), outIdx(0),
+        testcaseName(testcaseName) {}
+
+    TopKParams(
+        const Tensor& A, const Tensor& k, const int64_t axis,
+        const op::v1::TopK::Mode mode, const op::v1::TopK::SortType sort,
+        const Tensor& result0, const Tensor& result1, size_t outIdx,
+        const std::string& testcaseName = "") :
+        A(A), k(k), axis(axis), mode(mode), sort(sort),
+        result0(result0), result1(result1), outIdx(outIdx),
+        testcaseName(testcaseName) {}
+
+    Tensor A;
+    Tensor k;
+    int64_t axis;
+    op::v1::TopK::Mode mode;
+    op::v1::TopK::SortType sort;
+    Tensor result0;
+    Tensor result1;
+    size_t outIdx;
+    std::string testcaseName;
+};
+
+template <typename T>
+void copy_data(runtime::Tensor& t, const std::vector<T>& data) {
```

Comment:
Why do we need this helper? Do we need this helper in other tests? If yes, should we move it to common utils?

Reply:
Removed. Thanks